### ISSUE TITLE
Latest 15 Papers - April 19, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#2]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Single-Shot Shape and Reflectance with Spatial Polarization Multiplexing](http://arxiv.org/abs/2504.13177v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13177v1) |  |
| **[Do Vision-Language Models Represent Space and How? Evaluating Spatial Frame of Reference Under Ambiguities](http://arxiv.org/abs/2410.17385v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2410.17385v2) | <details><summary>Accep...</summary><p>Accepted to ICLR 2025 (Oral) | Project page: https://spatial-comfort.github.io/</p></details> |
| **[Spatial Confidence Regions for Excursion Sets with False Discovery Rate Control](http://arxiv.org/abs/2504.13124v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13124v1) |  |
| **[VistaDPO: Video Hierarchical Spatial-Temporal Direct Preference Optimization for Large Video Models](http://arxiv.org/abs/2504.13122v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13122v1) | <details><summary>Code ...</summary><p>Code and Data: https://github.com/HaroldChen19/VistaDPO</p></details> |
| **[Benchmarking the Spatial Robustness of DNNs via Natural and Adversarial Localized Corruptions](http://arxiv.org/abs/2504.01632v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.01632v2) | Under review |
| **[Real-time High-fidelity Gaussian Human Avatars with Position-based Interpolation of Spatially Distributed MLPs](http://arxiv.org/abs/2504.12909v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12909v1) | CVPR 2025 |
| **[Spatial Functional Deep Neural Network Model: A New Prediction Algorithm](http://arxiv.org/abs/2504.12750v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12750v1) | <details><summary>33 pa...</summary><p>33 pages, 7 figures, 3 tables</p></details> |
| **[Listen to Your Map: An Online Representation for Spatial Sonification](http://arxiv.org/abs/2412.05486v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2412.05486v2) |  |
| **[SmartFreeEdit: Mask-Free Spatial-Aware Image Editing with Complex Instruction Understanding](http://arxiv.org/abs/2504.12704v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12704v1) |  |
| **[Embodied-R: Collaborative Framework for Activating Embodied Spatial Reasoning in Foundation Models via Reinforcement Learning](http://arxiv.org/abs/2504.12680v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12680v1) | 12 pages, 5 figures |
| **[Can Moran Eigenvectors Improve Machine Learning of Spatial Data? Insights from Synthetic Data Validation](http://arxiv.org/abs/2504.12450v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12450v1) |  |
| **[Geographical Context Matters: Bridging Fine and Coarse Spatial Information to Enhance Continental Land Cover Mapping](http://arxiv.org/abs/2504.12368v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12368v1) |  |
| **[GripMap: An Efficient, Spatially Resolved Constraint Framework for Offline and Online Trajectory Planning in Autonomous Racing](http://arxiv.org/abs/2504.12115v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12115v1) |  |
| **[Single-shot Star-convex Polygon-based Instance Segmentation for Spatially-correlated Biomedical Objects](http://arxiv.org/abs/2504.12078v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12078v1) | 12 pages, 8 figures |
| **[Local Clustering and Global Spreading of Receptors for Optimal Spatial Gradient Sensing](http://arxiv.org/abs/2410.03395v2)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2410.03395v2) | <details><summary>This ...</summary><p>This version has been accepted for publication in Physical Review Letters. The final version is available at https://doi.org/10.1103/PhysRevLett.134.158401. Title has changed</p></details> |

## Spatio
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Spatio-Temporal Fuzzy-oriented Multi-Modal Meta-Learning for Fine-grained Emotion Recognition](http://arxiv.org/abs/2412.13541v3)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2412.13541v3) | <details><summary>This ...</summary><p>This work has been submitted to the IEEE for possible publication</p></details> |
| **[Multimodal Spatio-temporal Graph Learning for Alignment-free RGBT Video Object Detection](http://arxiv.org/abs/2504.11779v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.11779v1) |  |
| **[SPOT: Spatio-Temporal Pattern Mining and Optimization for Load Consolidation in Freight Transportation Networks](http://arxiv.org/abs/2504.09680v1)** | 2025-04-13 | [papers-cool](https://papers.cool/arxiv/2504.09680v1) |  |
| **[OmniMamba4D: Spatio-temporal Mamba for longitudinal CT lesion segmentation](http://arxiv.org/abs/2504.09655v1)** | 2025-04-13 | [papers-cool](https://papers.cool/arxiv/2504.09655v1) | <details><summary>Accep...</summary><p>Accepted at IEEE International Symposium on Biomedical Imaging (ISBI) 2025</p></details> |
| **[VideoChat-R1: Enhancing Spatio-Temporal Perception via Reinforcement Fine-Tuning](http://arxiv.org/abs/2504.06958v3)** | 2025-04-13 | [papers-cool](https://papers.cool/arxiv/2504.06958v3) |  |
| **[Efficient Spatio-Temporal Signal Recognition on Edge Devices Using PointLCA-Net](http://arxiv.org/abs/2411.14585v2)** | 2025-04-13 | [papers-cool](https://papers.cool/arxiv/2411.14585v2) | <details><summary>Accep...</summary><p>Accepted to International Joint Conference on Neural Networks(IJCNN), 2015</p></details> |
| **[DG-STMTL: A Novel Graph Convolutional Network for Multi-Task Spatio-Temporal Traffic Forecasting](http://arxiv.org/abs/2504.07822v2)** | 2025-04-11 | [papers-cool](https://papers.cool/arxiv/2504.07822v2) |  |
| **[Multi-resolution filters via linear projection for large spatio-temporal datasets](http://arxiv.org/abs/2401.05315v3)** | 2025-04-11 | [papers-cool](https://papers.cool/arxiv/2401.05315v3) | <details><summary>44 pa...</summary><p>44 pages, 10 figures, 2 tables</p></details> |
| **[SpaceVLLM: Endowing Multimodal Large Language Model with Spatio-Temporal Video Grounding Capability](http://arxiv.org/abs/2503.13983v3)** | 2025-04-11 | [papers-cool](https://papers.cool/arxiv/2503.13983v3) |  |
| **[Deep spatio-temporal point processes: Advances and new directions](http://arxiv.org/abs/2504.06364v1)** | 2025-04-08 | [papers-cool](https://papers.cool/arxiv/2504.06364v1) |  |
| **[STNAGNN: Data-driven Spatio-temporal Brain Connectivity beyond FC](http://arxiv.org/abs/2406.12065v2)** | 2025-04-08 | [papers-cool](https://papers.cool/arxiv/2406.12065v2) |  |
| **[STPro: Spatial and Temporal Progressive Learning for Weakly Supervised Spatio-Temporal Grounding](http://arxiv.org/abs/2502.20678v2)** | 2025-04-05 | [papers-cool](https://papers.cool/arxiv/2502.20678v2) | CVPR'25 Conference |
| **[DP-LET: An Efficient Spatio-Temporal Network Traffic Prediction Framework](http://arxiv.org/abs/2504.03792v1)** | 2025-04-04 | [papers-cool](https://papers.cool/arxiv/2504.03792v1) | 6 pages |
| **[Distributionally Robust Predictive Runtime Verification under Spatio-Temporal Logic Specifications](http://arxiv.org/abs/2504.02964v1)** | 2025-04-03 | [papers-cool](https://papers.cool/arxiv/2504.02964v1) | <details><summary>arXiv...</summary><p>arXiv admin note: substantial text overlap with arXiv:2311.09482</p></details> |
| **[Towards Multi-agent Reinforcement Learning based Traffic Signal Control through Spatio-temporal Hypergraphs](http://arxiv.org/abs/2404.11014v2)** | 2025-04-03 | [papers-cool](https://papers.cool/arxiv/2404.11014v2) | <details><summary>Accep...</summary><p>Accepted by IEEE Transactions on Mobile Computing</p></details> |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Sleep-time Compute: Beyond Inference Scaling at Test-time](http://arxiv.org/abs/2504.13171v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13171v1) | <details><summary>Code ...</summary><p>Code and data released at: https://github.com/letta-ai/sleep-time-compute</p></details> |
| **[A New Semidefinite Relaxation for Linear and Piecewise-Affine Optimal Control with Time Scaling](http://arxiv.org/abs/2504.13170v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13170v1) |  |
| **[Propagation of Chaos in One-hidden-layer Neural Networks beyond Logarithmic Time](http://arxiv.org/abs/2504.13110v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13110v1) | 70 pages |
| **[Efficient Adaptive Bandwidth Allocation for Deadline-Aware Online Admission Control in Time-Sensitive Networking](http://arxiv.org/abs/2503.09093v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2503.09093v2) |  |
| **[Risk Estimate under a Time-Varying Autoregressive Model for Data-Driven Reproduction Number Estimation](http://arxiv.org/abs/2409.14937v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2409.14937v2) |  |
| **[Real-time High-fidelity Gaussian Human Avatars with Position-based Interpolation of Spatially Distributed MLPs](http://arxiv.org/abs/2504.12909v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12909v1) | CVPR 2025 |
| **[Test-time Alignment of Diffusion Models without Reward Over-optimization](http://arxiv.org/abs/2501.05803v3)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2501.05803v3) | <details><summary>ICLR ...</summary><p>ICLR 2025 (Spotlight). The Thirteenth International Conference on Learning Representations. 2025</p></details> |
| **[Nash equilibria in scalar discrete-time linear quadratic games](http://arxiv.org/abs/2410.12544v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2410.12544v2) | <details><summary>Updat...</summary><p>Updated based on the reviews from ECC25. Camera ready version</p></details> |
| **[Timing via Pinwheel Double Categories](http://arxiv.org/abs/2504.12846v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12846v1) | <details><summary>10 pa...</summary><p>10 pages, uses formulations from 'Monoidal Context Theory' (arXiv:2404.06192) and 'String Diagrams for Physical Duoidal Categories' (arXiv:2406.19816)</p></details> |
| **[ALT: A Python Package for Lightweight Feature Representation in Time Series Classification](http://arxiv.org/abs/2504.12841v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12841v1) | 16 pages, 4 figures |
| **[LSEAttention is All You Need for Time Series Forecasting](http://arxiv.org/abs/2410.23749v6)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2410.23749v6) | <details><summary>8 pag...</summary><p>8 pages with referencing, 1 figure, 5 tables</p></details> |
| **[Control the GNN: Utilizing Neural Controller with Lyapunov Stability for Test-Time Feature Reconstruction](http://arxiv.org/abs/2410.09708v2)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2410.09708v2) | <details><summary>This ...</summary><p>This work has been submitted to the IEEE for possible publication</p></details> |
| **[Biasing the Driving Style of an Artificial Race Driver for Online Time-Optimal Maneuver Planning](http://arxiv.org/abs/2504.12744v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12744v1) |  |
| **[TimeCapsule: Solving the Jigsaw Puzzle of Long-Term Time Series Forecasting with Compressed Predictive Representations](http://arxiv.org/abs/2504.12721v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12721v1) |  |
| **[Accurate Tracking of Arabidopsis Root Cortex Cell Nuclei in 3D Time-Lapse Microscopy Images Based on Genetic Algorithm](http://arxiv.org/abs/2504.12676v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12676v1) |  |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Uncertainty-Aware Trajectory Prediction via Rule-Regularized Heteroscedastic Deep Classification](http://arxiv.org/abs/2504.13111v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13111v1) | <details><summary>17 Pa...</summary><p>17 Pages, 9 figures. Accepted to Robotics: Science and Systems(RSS), 2025</p></details> |
| **[Set You Straight: Auto-Steering Denoising Trajectories to Sidestep Unwanted Concepts](http://arxiv.org/abs/2504.12782v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12782v1) | Preprint |
| **[Trajectory Adaptation using Large Language Models](http://arxiv.org/abs/2504.12755v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12755v1) | <details><summary>Accep...</summary><p>Accepted to CoRL LangRob workshop 2024</p></details> |
| **[Distribution Backtracking Builds A Faster Convergence Trajectory for Diffusion Distillation](http://arxiv.org/abs/2408.15991v3)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2408.15991v3) | <details><summary>Our c...</summary><p>Our code is publicly available on https://github.com/SYZhang0805/DisBack</p></details> |
| **[TraCeS: Trajectory Based Credit Assignment From Sparse Safety Feedback](http://arxiv.org/abs/2504.12557v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.12557v1) |  |
| **[Adapting a World Model for Trajectory Following in a 3D Game](http://arxiv.org/abs/2504.12299v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12299v1) |  |
| **[GripMap: An Efficient, Spatially Resolved Constraint Framework for Offline and Online Trajectory Planning in Autonomous Racing](http://arxiv.org/abs/2504.12115v1)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.12115v1) |  |
| **[Stochastic Trajectory Optimization for Robotic Skill Acquisition From a Suboptimal Demonstration](http://arxiv.org/abs/2408.03131v3)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2408.03131v3) |  |
| **[Trajectory-guided Motion Perception for Facial Expression Quality Assessment in Neurological Disorders](http://arxiv.org/abs/2504.09530v2)** | 2025-04-16 | [papers-cool](https://papers.cool/arxiv/2504.09530v2) | <details><summary>Accep...</summary><p>Accepted to IEEE FG 2025 (preprint)</p></details> |
| **[Trajectory Encoding Temporal Graph Networks](http://arxiv.org/abs/2504.11386v1)** | 2025-04-15 | [papers-cool](https://papers.cool/arxiv/2504.11386v1) |  |
| **[GC-GAT: Multimodal Vehicular Trajectory Prediction using Graph Goal Conditioning and Cross-context Attention](http://arxiv.org/abs/2504.11150v1)** | 2025-04-15 | [papers-cool](https://papers.cool/arxiv/2504.11150v1) |  |
| **[Towards Robust Trajectory Embedding for Similarity Computation: When Triangle Inequality Violations in Distance Metrics Matter](http://arxiv.org/abs/2504.10933v1)** | 2025-04-15 | [papers-cool](https://papers.cool/arxiv/2504.10933v1) | 14 pages, 8 figures |
| **[Leveraging Driver Field-of-View for Multimodal Ego-Trajectory Prediction](http://arxiv.org/abs/2312.08558v2)** | 2025-04-15 | [papers-cool](https://papers.cool/arxiv/2312.08558v2) | <details><summary>Accep...</summary><p>Accepted to 13th International Conference on Learning Representations (ICLR 2025), 29 pages</p></details> |
| **[RINGO: Real-time Navigation with a Guiding Trajectory for Aerial Manipulators in Unknown Environments](http://arxiv.org/abs/2504.08338v2)** | 2025-04-14 | [papers-cool](https://papers.cool/arxiv/2504.08338v2) | 9 pages, 15 figures |
| **[Transferable text data distillation by trajectory matching](http://arxiv.org/abs/2504.09818v1)** | 2025-04-14 | [papers-cool](https://papers.cool/arxiv/2504.09818v1) |  |
